my.css

```diff
@@ -352,3 +352,166 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+.bagscluth__content {
+  display: flex;
+  max-width: 1150px;
+  width: 100%;
+  margin: 0 auto;
+}
+.bagscluth {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  max-height: 900px;
+  display: flex;
+  background-color: #fae8e4;
+}
+.bagscluth__img {
+  width: 60%;
+  height: 844px;
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)